src/dev_config_schema.yaml

```diff
@@ -0,0 +1,144 @@
+global:
+  # Logging settings
+  log_to_file: 
+    top_level: False
+    bottom_level: True
```

Comment:
If top_level and bottom_level are mutually exclusive, could we just do with one of them? I'd suggest bottom_level is the important one?

Reply:
Its better practice in my opinion to limit optional config options to be datatype specific. E.g., min and max for a float/int and file extension for a path.

Reply:
Changed this to one config option - 'singular'